cmd/receptor.go

```diff
@@ -95,6 +95,7 @@ func (cfg nullBackendCfg) Run() error {
 }
 
 func main() {
+	controlsvc.Version = version
```

Comment:
Maybe what we want to do here is have a new package that holds the version resource, instead of having it be in main, so that other packages can import it without dependency loops?